.github/actions/build_release/action.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Build Release'
+description: 'Build with release profile'
+inputs:
+  github_token:
+    description: "Github Token"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: arduino/setup-protoc@v1
+      with:
+        version: '3.x'
+        repo-token: ${{ inputs.github_token }}
+
+    - uses: actions-rs/toolchain@v1
+      with:
+        target: ${{ matrix.config.target }}
+        components: rustfmt, clippy
+
+    - uses: Swatinem/rust-cache@v1
+
+    - name: Build Databend
+      uses: actions-rs/cargo@v1
+      with:
+        command: build
+        use-cross: ${{ matrix.config.cross }}
+        args: --release --target ${{ matrix.config.target }}
```

Comment:
Need to control the size of the release on `ubuntu-latest`. I think it may be necessary to set `RUSTFLAGS` as a separate step up front.

Reply:
How about moving to another PR to address this?

Reply:
I will continue work in issue https://github.com/datafuselabs/databend/issues/3382

Reply:
ok